src/main/kotlin/features/uuid/UuidCommand.kt

```diff
@@ -5,15 +5,21 @@ import com.github.ajalt.clikt.core.Context
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.validate
+import com.github.ajalt.clikt.parameters.types.enum
 import com.github.ajalt.clikt.parameters.types.int
 import org.jack.features.uuid.services.UuidService
 
+enum class UuidType { UUID, ULID }
+
 const val UUID_COMMAND_NAME = "uuid"
 const val UUID_HELP = "Generate a random unique identifier"
 const val UUID_COUNT_NAME = "--count"
 const val UUID_COUNT_NAME_SHORT = "-c"
 const val UUID_COUNT_HELP = "Number of unique identifiers"
 const val UUID_COUNT_MUST_BE_POSITIVE_INTEGER = "Must be a positive integer"
+const val UUID_TYPE_NAME = "--type"
+const val UUID_TYPE_NAME_SHORT = "-t"
+const val UUID_TYPE_HELP = "The type of the unique identifier."
```

Comment:
Nitpick. Just so we follow the same conventions:

```suggestion
const val UUID_TYPE_HELP = "The type of the unique identifier"
```

Reply:
Changed, will resolve